frontend/tests/acceptance/features/webUIDashboard/dashboard.feature

```diff
@@ -0,0 +1,74 @@
+      | Office Manager | ABC Company | Medium   | Yet to Apply  | Applied       |
+      | Office Manager | ABC Company | Medium   | Applied       | In Progress   |
+      | Office Manager | ABC Company | Medium   | In Progress   | Accepted      |
+      | Office Manager | ABC Company | Medium   | Accepted      | Rejected      |
+      | Office Manager | ABC Company | Medium   | Rejected      | Yet to Apply  |
+
+  @movewithin
+  Scenario Outline: Move job application within a status board
+		Given the following job application already exists:
+      | title          | company     | priority   | status   |
+      | Office Manager | ABC Company | Medium     | <status> |
+      | Sales Manager  | XYZ Company | Low        | <status> |
+    When the user moves second job application above the first job application within "<status>" status board
+    Then the second job applications should appear above the first job application within "<status>" status board
+    Examples:
+      | status       |
+      | Yet to Apply |
+      | Applied      |
+      | In Progress  |
+      | Accepted     |
+      | Rejected     |
+    
+  @deletejob
+  Scenario Outline: Delete job application from dashboard
+		Given the following job application already exists:
+      | title   | company   | priority   | status   |
+      | <title> | <company> | <priority> | <status> |
+    When the user deletes job application of "<status>" status board using webUI
+    Then the job application of title "<title>" should not appear in "<status>" status board
+    Examples:
+      | title          | company     | priority | status       |
+      | Office Manager | ABC Company | Medium   | Yet to Apply |
+      | Office Manager | ABC Company | Medium   | Applied      |
+      | Office Manager | ABC Company | Medium   | In Progress  |
+      | Office Manager | ABC Company | Medium   | Accepted     |
+      | Office Manager | ABC Company | Medium   | Rejected     |
+  
+  @gotoupdatejob
+  Scenario Outline: Go to update job application page from dashboard
+		Given the following job application already exists:
+      | title   | company   | priority   | status   |
+      | <title> | <company> | <priority> | <status> |
+    When the user clicks job application of title "<title>" from "<status>" status board
+    Then the user should be redirected to update job application page and should see following data:
+      | title   | company   | priority   | status   |
+      | <title> | <company> | <priority> | <status> |
+    Examples:
+      | title          | company     | priority | status       |
+      | Office Manager | ABC Company | Medium   | Yet to Apply |
+      | Office Manager | ABC Company | Medium   | Applied      |
+      | Office Manager | ABC Company | Medium   | In Progress  |
+      | Office Manager | ABC Company | Medium   | Accepted     |
+      | Office Manager | ABC Company | Medium   | Rejected     |
```

Comment:
same as above.
`title`, `Company`, and `priority` have a single value in all rows. You do not need to keep it in the Examples table.